pkg/model/sdk_helper.go

```diff
@@ -254,6 +254,9 @@ func getMemberByPath(
 	elements := strings.Split(path, ".")
```

Comment:
Might be worth calling out in the docstring that `..` notation for list/map types is currently ignored.

Reply:
sure, i will add the comment based on the discussion on the above comment.

Reply:
Added a comment and TODO.